src/base/rss/rss_autodownloadrule.cpp

```diff
@@ -41,50 +41,49 @@
 #include <QStringList>
 
 #include "../preferences.h"
-#include "../tristatebool.h"
 #include "../utils/fs.h"
 #include "../utils/string.h"
 #include "rss_feed.h"
 #include "rss_article.h"
 
 namespace
 {
-    TriStateBool jsonValueToTriStateBool(const QJsonValue &jsonVal)
+    boost::optional<bool> jsonValueToOptional(const QJsonValue &jsonVal)
```

Comment:
jsonValueToOptionalBool or just jsonValueToBool

Reply:
Done (and elsewhere)